src/stores/QuickNavigationStore.js

```diff
@@ -10,7 +10,7 @@
 
 export default {
   state: {
-    enableQuickNavigation: false,
+    showQuickNavigation: false,
```

Comment:
wont this change break the current functionality?

Reply:
yes, @marinaaisa this has to stay as `enableQuickNavigation ` for the modal to appear, or refactor the appearances of `enableQuickNavigation` and change it to `showQuickNavigation `. Rn with this implementation when the magnifier icon is clicked nothing will happen bc the `modal-open` class is applied when `enableQuickNavigation` is set to true in `DocumentationTopicView` component.

Reply:
I just fixed it at the same time :D c3ebcd9c2a1a3e0dd20780f944092241f6f2128e

Reply:
Is working properly @sofiaromorales ? I'm trying out and it does for me. Let me know if I'm missing something else

Reply:
@marinaaisa yeap the code looks good to me, how big was the index file that was causing the app to break?

Reply:
Very big :)
We should test with big amount of data too so we make sure this is a solid feature for all kinds of documentation.
I will try to provide you with a big OSS data example soon!